content/en/docs/tutorials/clusters/seccomp.md

```diff
@@ -0,0 +1,6 @@
+---
+reviewers:
+# - TODO 
+title: Seccomp
+content_template: templates/tutorial
+---
```

Comment:
Early feedback: This would work better as a Task page, I think.

Reply:
Gotcha! Thanks :) I had just put it here because the `AppArmor` one was here as well, but I tend to agree with you